src/UnwrapWritableStream.php

```diff
@@ -153,7 +153,7 @@ public function close()
         $this->closed = true;
 
         // try to cancel promise once the stream closes
-        if ($this->promise instanceof CancellablePromiseInterface) {
+        if ($this->promise instanceof CancellablePromiseInterface  || (method_exists($this->promise, 'cancel') && $this->promise instanceof PromiseInterface)) {
```

Comment:
Looks like a leftover, same as above?

Reply:
Yup, already made the change, but it somehow didn't get committed 0_o

Reply:
Looks like there's still a leftover `use CancellablePromiseInterface` in this file now?

Reply:
Not anymore :+1: